pkg/cosign/keys_test.go

```diff
@@ -48,7 +130,66 @@ func TestLoadECDSAPrivateKey(t *testing.T) {
 	if _, err := rand.Read(buf[:]); err != nil {
 		t.Fatal(err)
 	}
-	if _, err := LoadECDSAPrivateKey(buf[:], []byte("wrong")); err == nil {
+	if _, err := LoadPrivateKey(buf[:], []byte("wrong")); err == nil {
 		t.Error("expected error decrypting key!")
 	}
 }
+
+func TestImportPrivateKey(t *testing.T) {
+	td := t.TempDir()
```

Comment:
You don't need explicit Remove's for testing's TempDir
https://cs.opensource.google/go/go/+/go1.17.3:src/testing/testing.go;l=925